td-agent/Rakefile

```diff
@@ -168,11 +168,28 @@ def tar_command
   end
 end
 
+def use_ruby3?
+  # Check whether ubuntu 22.04 (jammy) or not
+  # Basically, TD Agent adopts ruby 2.7 as default version, but on Ubuntu 22.04,
+  # use ruby 3.x because distribution switches to OpenSSL 3.x.
+  return false unless File.exist?("/etc/os-release")
+  os_release_entries = File.open("/etc/os-release").readlines.map(&:strip)
+  os_release_entries.include?("UBUNTU_CODENAME=jammy") and os_release_entries.include?("ID=ubuntu")
```

Comment:
Nitpick: This is not required but also CentOS Stream 9 (and RockyLinux 9) will be needed to switch using Ruby 3.1.
This is just for you information but we will need to handle this Ruby 3.1 switching in the future.